### PR TITLE
Update backend docs

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

File: source/services/notifications.md [hunk 1]
```diff
@@ -1,18 +1,21 @@
 # Notification Service
 Allows users to send signed transaction messages between devices taking part in the signing process.
 
-[Show on GitHub](https://github.com/gnosis/safe-notification-service)
-
-[Show on Swagger](https://safe-notification.gnosis.pm/)
+[GitHub](https://github.com/gnosis/safe-notification-service)
+[Swagger](https://safe-notification.gnosis.io/)
```
Comment (tschubotz) [inline source/services/notifications.md]: Should we add the rinkeby link here as well?

Comment (tschubotz) [reply source/services/notifications.md]: same like for the relay below :)

File: source/services/relay.md [hunk 2]
```diff
@@ -22,126 +23,141 @@ Our target user hold crypto in a centralized exchange (or on another Ethereum ad
...
+
+If all checks pass, then Safe address is valid and the user can transfer at least the `payment` amount of ETH (if `paymentToken` is address `0x0`) or the corresponding amount of `paymentToken` tokens (if the `paymentToken` is a valid token address) to the Safe address. Please take a look at the [api/v1/tokens/?gas=True](relay.html#tokens-get) endpoint to see which tokens are accepted for payment by our service.
+
+Otherwise, the response is not valid or it is compromised, and it should not be used any further.
 
-Otherwise, the response has error or it is compromised, and it should not be used any further.
-
-----------
+In the context of the Gnosis Safe and our relay services:
+- The `MasterCopy`, `ProxyFactory` and `FallbackHandler` should be addresses of a valid deployment of the `GnosisSafe` contracts.
```
Comment (tschubotz) [inline source/services/relay.md]: Is there somewhere a mention of the supported master copies,proxcy factoryies and fallbackhanders? if not, should we add it?

File: source/services/notifications.md (hunk 1 above)
Comment (Uxio0) [reply source/services/notifications.md]: Agree :)

File: source/services/relay.md (hunk 2 above)
Comment (Uxio0) [reply source/services/relay.md]: I added one here https://github.com/gnosis/safe-docs/pull/23/files#diff-b3e28734e1cd5ff2616bc1444cb004b1R71, but yes, it won't harm to put that more visible

File: source/services/notifications.md (hunk 1 above)
Comment (Uxio0) [reply source/services/notifications.md]: I checked an relay already has the link for both and notification service is the same for rinkeby/mainnet

Comment (Uxio0) [reply source/services/notifications.md]: I added Releases for Notification Service

**Pull request merged**